### PR TITLE
Payroll: Add tasks for Travis

### PR DESCRIPTION
**Pull request by facuspagnuolo**
Adding tests and coverage tasks to travis config file

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -29,12 +30,14 @@ env:
...
     - TASK=test:vault
     - TASK=test:voting
+    - TASK=test:payroll
     - TASK=coverage:agent
     - TASK=coverage:finance
     - TASK=coverage:survey
     - TASK=coverage:token-manager
     - TASK=coverage:vault
     - TASK=coverage:voting
+    - TASK=coverage:payroll
```
Comment (sohkai) [inline .travis.yml]: Ahh, I think we need to add these two tasks to the package.json.

**Comment by sohkai** [comment]
@facuspagnuolo Oops, forgot we weren't bootstrapping the future apps before; this should be fixed now :).

**Comment by facuspagnuolo** [comment]
Thanks @sohkai ! I didn't understand why it wasn't working, thanks :)

**Comment by facuspagnuolo** [comment]
Closing in favor of #754

**Pull request closed without merging**